src/Microsoft.AspNetCore.BuildTools.ApiCheck/build/Microsoft.AspNetCore.BuildTools.ApiCheck.targets

```diff
@@ -0,0 +1,30 @@
+﻿<!--
```

Comment:
Why is that odd distance between tasks and where they are used a "design pattern"?